python/ray/autoscaler/_private/gcp/node_provider.py

```diff
@@ -203,16 +202,20 @@ def _thread_unsafe_terminate_node(self, node_id: str):
     @_retry
     def terminate_node(self, node_id: str):
         with self.lock:
-            self._thread_unsafe_terminate_node(node_id)
-
-    def terminate_nodes(self, node_ids: List[str]):
-        if not node_ids:
-            return None
-
-        with self.lock, concurrent.futures.ThreadPoolExecutor() as executor:
-            result = executor.map(self._thread_unsafe_terminate_node, node_ids)
-
-        return list(result)
+            resource = self._get_resource_depending_on_node_name(node_id)
```

Comment:
This simply reverts https://github.com/ray-project/ray/commit/46fc66390cb136a52afd6d70808f7c01eb31deb0